src/contractor/contractor.cpp

```diff
@@ -95,10 +98,11 @@ EdgeWeight getNewWeight(IterType new_speed_iter, const double segment_length, Ed
     {
         auto newSecs = new_segment_weight / 10;
         auto oldSecs = oldWeight / 10;
+        auto speed_file = segment_speed_filenames.at(new_speed_iter->speed_source.source - 1);
```

Comment:
So is it guaranteed that we can never loose coverage for an edge? So if a speed ever gets accessed here, we will never put in an update that would fall back to `lua` for this edge?